policyengine_us/parameters/gov/states/oh/odjfs/owf/payment_standard/amounts.yaml

```diff
@@ -0,0 +1,31 @@
+description: Ohio sets this monthly payment standard for Ohio Works First program.
```

Comment:
Just note, no action needed:

I think the monthly vs yearly definition is displayed on the frontend, at least it was in App V1